### DISCUSSION TITLE
What is the total number of points in Scramble?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#969] [Galaxy Trivia] [answered]
**Bonus:** Who invented the game?

**Comment by peterbaccaro** [comment, marked as answer]
The total face point value is **187**. 

There are 225 squares on a Scrabble board. Point value is as follows:
- 1 point - A, E, I, O, U, L, N, S, T, R
- 2 points - D, G
- 3 points - B, C, M, P
- 4 points - F, H, V, W, Y
- 5 points - K
- 8 points - J, X
- 10 points - Q, Z
- 0 points - Blank Tiles 

Scrabble letter distribution is as follows: A-9, B-2, C-2, D-4, E-12, F-2, G-3, H-2, I-9, J-1, K-1, L-4, M-2, N-6, O-8, P-2, Q-1, R-6, S-4, T-6, U-4, V-2, W-2, X-1, Y-2, Z-1 and Blanks-2.

Scrabble was invented by **Alfred Mosher Butts**, an American architect, in the 1930s. Butts initially called the game "Lexiko" and later renamed it "Criss-Cross Words." The game eventually evolved into the Scrabble we know today, with the trademark being registered in 1948. James Brunot acquired the rights to the game in 1948 and helped popularize it further. Since then, Scrabble has become one of the most popular word games worldwide.
